src/ric/1AC60.c

```diff
@@ -342,7 +342,131 @@ void func_80158FA4(void) {
     }
 }
 
-INCLUDE_ASM("asm/us/ric/nonmatchings/1AC60", func_801590A0);
+void func_801590A0(void) {
+    if ((D_80154570 == 0) && (PLAYER.velocityY < FIX(-1)) &&
+        !(g_Player.unk44 & 0x40) && !(g_Player.padPressed & PAD_CROSS)) {
+        PLAYER.velocityY = FIX(-1);
+    }
+    if ((g_Player.pl_vram_flag & 2) && (PLAYER.velocityY < FIX(-1))) {
+        PLAYER.velocityY = FIX(-0.25);
+        g_Player.unk44 |= 0x20;
```

Comment:
unk44 seems to be bit flags, could start towards documenting it, but not essential

Reply:
Agreed, I think that was actually why I started decompiling this function originally. I found `g_Player.unk44 & 1` seems to be the flag for being able to do a dive-kick and wanted to find any other uses to finish documenting it. So maybe I'll look for more uses of that variable to decompile.